packages/idealo.design/scripts/motifuiImporter.js

```diff
@@ -116,10 +116,16 @@ async function createFormDataForComponents(components) {
 }
 
 async function sendDataToHttpRequest(components) {
+    const username = process.env.USER_NAME
+    const password = process.env.PASSWORD
     for (const component of components){
         const componentFormData = component.formData
         await axios.put(process.env.BASE_URL + '/api/components/update', componentFormData, {
-            headers: componentFormData.getHeaders()
+            headers: {
+                ...componentFormData.getHeaders(),
+                username: username,
+                password: password
```

Comment:
for basic auth with axios you can use this request config option: https://github.com/axios/axios#request-config (search for `auth: {`)